sfft_and_animate.py

```diff
@@ -80,7 +80,7 @@ def sfft_and_animate(oid,band):
     # First, need all the images that don't contain the SN. 
     out_tab = get_mjd_info(start,end,return_inverse=True)
     out_rows = np.where(np.isin(tab['pointing'],out_tab['pointing']))[0]
-    out_tab = tab[out_rows][:10] # Only use the first 5 images 
+    out_tab = tab[out_rows][:5] # Only use the first 5 images 
```

Comment:
Why 5, or 10 images?

Reply:
Ah, I took it down to 5 for speed while testing. No particular logic.